WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -177,4 +177,12 @@ def duration_to_s(total_seconds)
 
     [hours > 0 ? "#{hours}h " : '', minutes > 0 ? "#{minutes}m " : '', format('%.2f', seconds), 's'].join
   end
+
+  def wca_id_link(wca_id, options = {})
+    if wca_id
+      content_tag :span, class: "wca-id" do
+        link_to wca_id, "#{root_url}results/p.php?i=#{wca_id}", options
```

Comment:
Mostly as a space saver. I agree that it's insignificant.

It also carries the risk that you make the wrong kind of link in an email.